packages/addons/service/docker/package.mk

```diff
@@ -3,14 +3,14 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="docker"
-PKG_VERSION="18.06.1"
-PKG_SHA256="153cb489033686260dfe7a42acbdd1753d56f7a9c2d7ad90633f0c8cce563b23"
-PKG_REV="119"
+PKG_VERSION="18.09.1"
+PKG_SHA256="9eadb1eae1954b0322aadf6505f5397d1b1eccf6395ab511cadf8e6975cfc576"
+PKG_REV="122"
```

Comment:
```suggestion
PKG_REV="120"
```

Reply:
and fix the changelog.txt ?

Reply:
I've made some Community builds to test older Docker versions so it be Rev 122 to make sure updating will work flawless.